src/components/text-input/__tests__/TextInput-test.tsx

```diff
@@ -94,7 +94,7 @@ it('should display an valid message when validation is valid', () => {
   const commonProps = { helpText: 'Helping', label: 'Label', messageValid: 'Youhou!' };
   const { rerender } = render(<TextInput {...commonProps} validation="none" />);
 
-  expect(screen.getByRole('textbox', { description: 'Helping' })).toBeInTheDocument();
+  expect(screen.getByRole('textbox', { description: ' Helping' })).toBeInTheDocument();
```

Comment:
The text input is described by both the validation message and the description, it seems like the empty validation message is adding a space before the description. I believe this is harmless, but I could be wrong.